SpatialGDK/Source/SpatialGDK/Private/Interop/RPCContainer.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Ah I think the best place for this file is inside the Utils folder. Sorry about the flip flopping.

Reply:
Moved to Utils.